/**:
  imu_converter_node:
    ros__parameters:
      input_topic: "IMUSensor"
      output_topic: "imu/data"

      override_covariance: true
      accel_noise_sigma: 0.0078 # [m/s^2] Standard deviation of accelerometer measurements.
      gyro_noise_sigma: 0.0012 # [rad/s] Standard deviation of gyroscope measurements.

  gps_converter_node:
    ros__parameters:
      input_topic: "GPSSensor"
      output_topic: "gps/fix"

      origin_latitude: 40.33940 # 40.23890
      origin_longitude: -111.90721 # -111.74212
      origin_altitude: 1412.0

      override_covariance: true
      position_noise_sigma: 0.015 # [m] Standard deviation of GPS position measurements.
      altitude_noise_sigma: 0.025 # [m] Standard deviation of GPS altitude measurements.

  depth_converter_node:
    ros__parameters:
      input_topic: "DepthSensor"
      output_topic: "odometry/depth"

      override_covariance: true
      noise_sigma: 0.02 # [m] Standard deviation of depth measurements.

  ahrs_converter_node:
    ros__parameters:
      input_topic: "RotationSensor"
      output_topic: "imu/ahrs"

      yaw_noise_sigma: 0.01745 # [rad] Standard deviation of yaw measurements.
      roll_pitch_noise_sigma: 0.00349 # [rad] Standard deviation of roll and pitch measurements.

  dvl_converter_node:
    ros__parameters:
      input_topic: "DVLSensorVelocity"
      output_topic: "dvl/twist"

      simulate_dropout: false
      dropout_frequency: 0.033333333 # [Hz] 1/30 seconds
      dropout_duration: 5.0 # [s]

      override_covariance: true
      noise_sigma: 0.02 # [m/s] Standard deviation of DVL body-frame velocity measurements.

  truth_converter_node:
    ros__parameters:
      input_topic: "DynamicsSensorOdom"
      output_topic: "odometry/truth"

      publish_tf: false # Whether to publish the map->base_link transform

  fin_state_publisher_node:
    ros__parameters:
      input_topic: "ControlCommand"
      output_topic: "joint_states"

  cmd_vel_converter_node:
    ros__parameters:
      input_topic: "cmd_vel"
      output_topic: "/command/agent"

  hsd_converter_node:
    ros__parameters:
      heading_topic: "heading"
      speed_topic: "speed"
      depth_topic: "depth"
      output_heading_topic: "/heading"
      output_speed_topic: "/speed"
      output_depth_topic: "/depth"

  mag_converter_node:
    ros__parameters:
      input_topic: "MagnetometerSensor"
      output_topic: "imu/mag"

      override_covariance: true
      noise_sigma: 0.003 # [normalized] Standard deviation of magnetometer measurements.
